### PR TITLE
Implement `$bitsAllClear`

### PR DESCRIPTION
**Pull request by w84thesun**

### CODE REVIEW

**Review by seeforschauer — COMMENTED** [review]
would you add tests for int64 and for wrong values (i.e. string)?

**Comment by w84thesun** [comment]
> would you add tests for int64 and for wrong values (i.e. string)?

Done

File: cmd/dance/main_unix.go [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+package main
+
+import (
+	"context"
+	"os/signal"
+
+	"golang.org/x/sys/unix"
+)
+
+func notifyAppTermination(parent context.Context) (context.Context, context.CancelFunc) {
```
Comment (AlekSi) [inline cmd/dance/main_unix.go:27]: That should be in a separate PR

**Review by AlekSi — COMMENTED** [review]
Tests fail

File: cmd/dance/main_unix.go (hunk 1 above)
Comment (w84thesun) [reply cmd/dance/main_unix.go:27]: Ok, I'll move it to another PR.

**Comment by w84thesun** [comment]
> > Done
> 
> Please re-request review in that case.
> 
> But also: tests fail.

Yup, I figure it out after I've added tests. I'm finishing https://github.com/FerretDB/FerretDB/issues/347 than will take a look at this.

**Review by AlekSi — COMMENTED** [review]
There are conflicts now

File: tests/ferret.yml [hunk 2]
```diff
@@ -5,7 +5,7 @@ results:
   common:
     stats:
       expected_skip: 1
-      expected_pass: 33
+      expected_pass: 3
```
Comment (AlekSi) [inline tests/ferret.yml:8]: That has to be a mistake

Comment (w84thesun) [reply tests/ferret.yml]: Yeah, there should be 38. Fixed it
